internal/store/build_result.go

```diff
@@ -233,7 +234,7 @@ func NewDeployInfo(iTarget model.ImageTarget, podSet PodSet) DeployInfo {
 	}
 
 	// Only return the pod if it matches our image.
-	if pod.ContainerImageRef == nil || !iTarget.Ref.Matches(pod.ContainerImageRef) {
+	if pod.ContainerImageRef == nil || !iTarget.DeploymentRef.Matches(pod.ContainerImageRef) {
```

Comment:
i think this is going to interact badly with Maia's changes?
https://github.com/windmilleng/tilt/pull/1337

Reply:
I think I've merged master in a way that maintains Maia's changes.